bundle/src/main/java/com/jfrog/ide/eclipse/log/Logger.java

```diff
@@ -43,14 +58,22 @@ public void error(String message, Throwable t) {
 
 	@Override
 	public void info(String message) {
-		ilog.log(new Status(Status.INFO, ID, "[INFO] " + message));
+		log(INFO, Status.INFO, "[INFO] ", message);
 	}
 
 	@Override
 	public void warn(String message) {
-		ilog.log(new Status(Status.WARNING, ID, "[WARN] " + message));
-		logToProblemsLogger(message, Status.WARNING);
+		log(WARN, Status.WARNING, "[WARN] ", message);
 	}
+	
+    private void log(int level, int status, String prefix, String message) {
+        if (logLevel <= level) {
+            ilog.log(new Status(status, ID, prefix + message));
+            if (status == Status.WARNING || status == Status.ERROR) {
+                logToProblemsLogger(message, status);
+            }
+        }
```

Comment:
```suggestion
        if (minLogLevel > level) {
             return 
        }
        ilog.log(new Status(status, ID, prefix + message));
        if (status == Status.WARNING || status == Status.ERROR) {
            logToProblemsLogger(message, status);
        }
```
1. Keep code flat if possiable
2. I would change the attribute name `logLevel` to `minLogLevel` for readability